app/konnect/dev-portal/access-and-approval/manage-teams.md

```diff
@@ -0,0 +1,208 @@
+
+{:.important}
+> **Important:** If you currently have developers in your {{site.konnect_short_name}}, you must configure developer teams before enabling Portal RBAC (which is disabled by default). If you enable Portal RBAC before configuring teams for your developers, it will prevent all developers from seeing any services in your Dev Portal. We recommend setting up your teams and permissions before enabling RBAC which will allow for a seamless transition: developers see what they're supposed to, instead of nothing at all.
+
+### Create an API Product
+
+{% navtabs %}
+{% navtab Konnect UI %}
+In {% konnect_icon runtimes %} [**Runtime Manager**](https://cloud.konghq.com/us/runtime-manager), select the **Default** runtime group and follow these steps:
+
+1. Select **Gateway Services** from the side navigation bar, then **New Gateway Service**.
+
+1. From the **Add a Gateway Service** dialog, enter the following to create a new service:
+    * **Name:** classic_pizzas
+    * **Upstream URL:** `http://mockbin.org`
+    * Use the defaults for the remaining fields.
+1. Click **Save**. 
+1. To create an API product for your service, navigate to **API Products** in the sidebar, click **Add API Product** and enter `Classic Pizzas` in the **Product Name** field.
+1. Click **New Version** from the Actions dropdown menu and enter `v1` in the **Version Name** field and then click **Create**.
+1. Click the **v1** version and then click **Link** to add a service.
+1. Select **default** from the **Select Runtime Group** dropdown menu, select **classic_pizzas** from the **Gateway Service** dropdown menu, and then click **Save**.
+
+{% endnavtab %}
+{% navtab API %}
+1. Create the service?
+1. Create the `Classic Pizzas` API product:
+```bash
+curl --request POST \
+  --url https://<region>.api.konghq.com/v2/api-products \
+  -- data '{
+    "id": "e7a4f6c0-5777-4a37-bcb9-2d37d3b362ee",
+    "name": "Classic Pizzas",
+    "description": "API product for classic pizzas",
+    "portal_ids": [],
+    "created_at": "2023-01-01T00:00:00.000Z",
+    "updated_at": "2023-01-01T00:00:00.000Z",
+    "labels": {
+        "env": "test"
+        }
+    }'
+```
+1. Create the `v1` version for the `Classic Pizzas` API product:
+```bash
+curl --request POST \
+  --url https://<region>.api.konghq.com/v2/api-products/<api-product-id>/product-versions \
+  -- data '{
+    "id": "9f5061ce-78f6-4452-9108-ad7c02821fd5",
+    "name": "v1",
+    "gateway_service": {
+        "runtime_group_id": "e4d9ebb1-26b4-426a-b00e-cb67044f3baf",
+        "id": "09b4786a-3e48-4631-8f6b-62d1d8e1a7f3"
+        },
+    "publish_status": "unpublished",
+    "deprecated": false,
+    "created_at": "2023-01-01T00:00:00.000Z",
+    "updated_at": "2023-01-01T00:00:00.000Z"
```

Comment:
No need to send `dreprecated`, `created_at`, or `updated_at` fields.  Also don't send `id` either